tests/examples/particle_tile_examples.py

```diff
@@ -0,0 +1,73 @@
+
+
+
+def test_ptile_data():
+    
+    ptd = amrex.ParticleTileData_1_1_2_1()
+    sp = amrex.Particle_3_2()
+    # sp.setPos([1,2,3])
+    sp.x = 1
+    # ptd.setSuperParticle(sp,0)
+    # ptd.getSuperParticle(0)
+    print(ptd.m_size)
+    print(ptd.m_num_runtime_real)
+    print(ptd.m_num_runtime_int)
+    pass
```

Comment:
asserts missing that validate the expected data.

Reply:
improved